contribs/osm/src/main/java/org/matsim/contrib/osm/networkReader/SupersonicOsmNetworkReader.java

```diff
@@ -391,6 +422,16 @@ private synchronized void addLinkToNetwork(Link link) {
             return this;
         }
 
+        /**
+         * This sets whether the free speed will be adjusted for urban links.
+         *
+         * @see LinkProperties#DEFAULT_FREESPEED_FACTOR
+         */
+        public AbstractBuilder<T> setAdjustFreeSpeed(boolean adjustFreeSpeed) {
```

Comment:
This is not necessary, or if one still wants to have the shortcut version for turning urban speed factor on and off could exchange the default ```afterLinkCreated``` with 
```
this.afterLinkCreated = ajdustFreeSpeed ? SupersonicOsmNetworkReader.adjustFreespeed(0.9) : SupersonicOsmNetworkReader.adjustFreespeed(01.0);
```
Though I'm not sure how to treat cases where the user has already attached an external ```afterLinkCreated``` Consumer